src/components/Query.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <div v-if="body" data-cy="query" class="mb-4">
-    <b>Query:</b>
+  <div class="bg-gray-200 rounded-sm p-2 mb-2" v-if="body" data-cy="query">
```

Comment:
we are reaching pixel-perfect design 😂 one more thing I noticed when using this boxed view, the `mb-2` class creates a bigger gap at the bottom of the box, so maybe going with something like `my-1` for all the boxes would work a little better
![Screenshot_2022-10-11_at_11_26_08](https://user-images.githubusercontent.com/23213553/195053057-2263bed8-0e4c-49a0-9d59-a2cef032fa2e.png)